targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Plugins/PlayFab/Source/PlayFab/Classes/PfTestActor.h.ejs

```diff
@@ -5,15 +5,15 @@
 #include "GameFramework/Actor.h"
 <% if (hasClientOptions) { %>
 #include "PlayFabClientModels.h"
-#include "PlayFabClientApi.h"
+#include "PlayFabClientAPI.h"
```

Comment:
This change in capitalization was required in order to get a successful Android build in 4.19.